setup.cfg

```diff
@@ -53,7 +53,7 @@ docs =
 tests =
     # Necessary for Windows until https://github.com/pytest-dev/pytest/issues/9765
     # is resolved:
-    pytest!=7.1.0, !=7.1.1;sys_platform=="win32"
+    pytest!=7.1.0, !=7.1.1, !=7.1.2;sys_platform=="win32"
```

Comment:
I am trying to track a failure in Jdaviz and I am suspecting it has something to do with `asdf`. Why did you pin a `!=7.1.2` here?

Reply:
Oh hmm, this only is for Windows... my failure is on Linux.

Reply:
Does this error ring any bell to you?

```
2022-05-25T20:01:04.0755535Z INTERNALERROR> Traceback (most recent call last):
2022-05-25T20:01:04.0756248Z INTERNALERROR>   File "<frozen importlib._bootstrap>", line 939, in _find_spec
2022-05-25T20:01:04.0756783Z INTERNALERROR> AttributeError: '_SixMetaPathImporter' object has no attribute 'find_spec'
```

Reply:
Is this for ASDF?

Reply:
Sorry for the noise. It's not ASDF. Something happened in the Python 3.10 that shipped with Actions runner.